src/components/analyser/details/Analysts.tsx

```diff
@@ -44,21 +44,21 @@ const useStyles = makeStyles(({ palette }: Theme) =>
     redDot: {
       height: '25px',
       width: '25px',
-      backgroundColor: '#cc0000',
+      backgroundColor: '#cc0000', // palette.error
```

Comment:
why in comments?

Reply:
the color is quite different from the palette. So I'm not sure.